server/src/main/java/org/elasticsearch/index/snapshots/IndexShardSnapshotStatus.java

```diff
@@ -467,9 +471,9 @@ public String toString() {
             + "stage="
             + stage
             + ", startTime="
-            + startTime
+            + startTimeMillis
             + ", totalTime="
-            + totalTime
+            + totalTimeMillis
```

Comment:
nit: could rename the labels here too

Reply:
Resolved in 9bea854496827b1a2199db71e1a87ed9bce8177d and also added units to the immutable copy object